appcelerator/titanium_mobile
# feat(android): target api level 30 by default

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28047

**Summary:**
Titanium builds now "target" Android 11 by default.

**Test:**
1. Create a classic project.
2. Build for Android.
3. Open "Finder" or "Windows Explorer" and go to your project folder.
4. Go to subdirectory: `./build/android/app/build/intermediates/merged_manifests/debug`
5. Open file "AndroidManifest.xml".
6. Verify XML attribute `android:targetSdkVersion` is set to `30`.
